### ISSUE TITLE
[Bug]: Error when opening/creating: partial not found

### ISSUE THREAD

**Issue by ogalbnafets** [ryanmitchell/ti-ext-mealtimes#6]
### What happened?

When editing or creating a mealtime, this error shows up:
![image](https://user-images.githubusercontent.com/70335936/155974589-166a3d99-62c3-4c32-850b-00487eab2d54.png)
Error log down below.
Freshly installed, nothing changed yet.

### What did you expect to happen?

I expect to be able to change the menu times.

### Version

Tasty: 3.3.2, Mealtimes: 1.13

### What browser are you seeing the problem on?

Firefox, Chrome, Safari

### Relevant log output

```shell
Igniter\Flame\Exception\SystemException: The partial 'extensions/thoughtco/mealtimes/partials/flexible' is not found.
in /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php:181
Stack trace:
#0 /var/www/stwh/tasty_demo/storage/framework/views/ee89782974610825bf1b8c72d0c60e2bf7b99901.php(7): Admin\Classes\BaseWidget->makePartial()
#1 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(227): include('/var/www/stwh/t...')
#2 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(189): Admin\Classes\BaseWidget->makeFileContent()
#3 /var/www/stwh/tasty_demo/app/admin/widgets/Form.php(268): Admin\Classes\BaseWidget->makePartial()
#4 /var/www/stwh/tasty_demo/storage/framework/views/3e79e2aee134db5842adbe9f7d1adabb505d04df.php(16): Admin\Widgets\Form->renderFieldElement()
#5 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(227): include('/var/www/stwh/t...')
#6 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(189): Admin\Classes\BaseWidget->makeFileContent()
#7 /var/www/stwh/tasty_demo/storage/framework/views/e3b1dda0b0161f802ce59e7046830ce74a646092.php(10): Admin\Classes\BaseWidget->makePartial()
#8 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(227): include('/var/www/stwh/t...')
#9 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(189): Admin\Classes\BaseWidget->makeFileContent()
#10 /var/www/stwh/tasty_demo/storage/framework/views/3cb987dd830c6f6a7d6f904eec5f826b4d0c15de.php(2): Admin\Classes\BaseWidget->makePartial()
#11 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(227): include('/var/www/stwh/t...')
#12 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(189): Admin\Classes\BaseWidget->makeFileContent()
#13 /var/www/stwh/tasty_demo/storage/framework/views/34e741d1852dbb0da6d0e2f55b986346f0580dbc.php(7): Admin\Classes\BaseWidget->makePartial()
#14 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(227): include('/var/www/stwh/t...')
#15 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(189): Admin\Classes\BaseWidget->makeFileContent()
#16 /var/www/stwh/tasty_demo/storage/framework/views/7faab0f8c5f375c9048eccb6d60064667dd1c112.php(2): Admin\Classes\BaseWidget->makePartial()
#17 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(227): include('/var/www/stwh/t...')
#18 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(189): Admin\Classes\BaseWidget->makeFileContent()
#19 /var/www/stwh/tasty_demo/storage/framework/views/685cebfc7b3d4b17d7fcee0e7187348888e23402.php(2): Admin\Classes\BaseWidget->makePartial()
#20 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(227): include('/var/www/stwh/t...')
#21 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(189): Admin\Classes\BaseWidget->makeFileContent()
#22 /var/www/stwh/tasty_demo/app/admin/widgets/Form.php(218): Admin\Classes\BaseWidget->makePartial()
#23 /var/www/stwh/tasty_demo/app/admin/actions/FormController.php(383): Admin\Widgets\Form->render()
#24 [internal function]: Admin\Actions\FormController->renderForm()
#25 /var/www/stwh/tasty_demo/vendor/tastyigniter/flame/src/Traits/ExtendableTrait.php(451): call_user_func_array()
#26 /var/www/stwh/tasty_demo/app/admin/classes/AdminController.php(539): Admin\Classes\AdminController->extendableCall()
#27 /var/www/stwh/tasty_demo/storage/framework/views/f55cfab22676aeab4d2f680c2ddf1b39307b284c.php(9): Admin\Classes\AdminController->__call()
#28 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(227): include('/var/www/stwh/t...')
#29 /var/www/stwh/tasty_demo/app/system/traits/ViewMaker.php(152): Admin\Classes\AdminController->makeFileContent()
#30 /var/www/stwh/tasty_demo/app/admin/classes/AdminController.php(287): Admin\Classes\AdminController->makeView()
#31 /var/www/stwh/tasty_demo/app/admin/classes/AdminController.php(223): Admin\Classes\AdminController->execPageAction()
#32 /var/www/stwh/tasty_demo/app/system/classes/Controller.php(117): Admin\Classes\AdminController->remap()
#33 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(54): System\Classes\Controller->runAdmin()
#34 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(45): Illuminate\Routing\Controller->callAction()
#35 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Routing/Route.php(262): Illuminate\Routing\ControllerDispatcher->dispatch()
#36 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Routing/Route.php(205): Illuminate\Routing\Route->runController()
#37 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Routing/Router.php(721): Illuminate\Routing\Route->run()
#38 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(128): Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#39 /var/www/stwh/tasty_demo/vendor/tastyigniter/flame/src/Translation/Middleware/Localization.php(27): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#40 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Igniter\Flame\Translation\Middleware\Localization->handle()
#41 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php(50): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#42 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Routing\Middleware\SubstituteBindings->handle()
#43 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#44 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\View\Middleware\ShareErrorsFromSession->handle()
#45 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(121): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#46 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(64): Illuminate\Session\Middleware\StartSession->handleStatefulRequest()
#47 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Session\Middleware\StartSession->handle()
#48 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#49 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle()
#50 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(67): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#51 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Cookie\Middleware\EncryptCookies->handle()
#52 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#53 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Routing/Router.php(723): Illuminate\Pipeline\Pipeline->then()
#54 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Routing/Router.php(698): Illuminate\Routing\Router->runRouteWithinStack()
#55 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Routing/Router.php(662): Illuminate\Routing\Router->runRoute()
#56 /var/www/stwh/tasty_demo/vendor/tastyigniter/flame/src/Router/CoreRouter.php(22): Illuminate\Routing\Router->dispatchToRoute()
#57 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(167): Igniter\Flame\Router\CoreRouter->dispatch()
#58 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(128): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}()
#59 /var/www/stwh/tasty_demo/vendor/tastyigniter/flame/src/Setting/Middleware/SaveSetting.php(27): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#60 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Igniter\Flame\Setting\Middleware\SaveSetting->handle()
#61 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#62 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TrimStrings.php(40): Illuminate\Foundation\Http\Middleware\TransformsRequest->handle()
#63 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Foundation\Http\Middleware\TrimStrings->handle()
#64 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#65 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Foundation\Http\Middleware\ValidatePostSize->handle()
#66 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php(86): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#67 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\Foundation\Http\Middleware\PreventRequestsDuringMaintenance->handle()
#68 /var/www/stwh/tasty_demo/vendor/fruitcake/laravel-cors/src/HandleCors.php(38): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#69 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Fruitcake\Cors\HandleCors->handle()
#70 /var/www/stwh/tasty_demo/vendor/fideloper/proxy/src/TrustProxies.php(57): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#71 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Fideloper\Proxy\TrustProxies->handle()
#72 /var/www/stwh/tasty_demo/extensions/igniter/api/vendor/dingo/api/src/Http/Middleware/Request.php(111): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#73 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Dingo\Api\Http\Middleware\Request->handle()
#74 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#75 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(142): Illuminate\Pipeline\Pipeline->then()
#76 /var/www/stwh/tasty_demo/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(111): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter()
#77 /var/www/stwh/tasty_demo/index.php(53): Illuminate\Foundation\Http\Kernel->handle()
#78 {main}
```

**Comment by ryanmitchell** [comment]
You need to rename the file .blade.php - you’ll see it just is .php at the moment.
If that works for you open a PR with the change and I’ll merge it in.

**Comment by ryanmitchell** [comment]
Fixed in 1.14
